### PR TITLE
Core/Spells: fix Improved Succubus warlock talent

### LINKED ISSUE #1543: [Warlock/Talents] Improved Succubus
this talent reduce the cast time of the spell Seduction that the your succubus use. 
the bug:
there is no any effect on the cast time. 0/3 , 1/3 , 2/3 , 3/3  - all of them same time 1.5 sec cast time. 

http://old.wowhead.com/spell=18754 [Improved Succubus - rank 1]
http://old.wowhead.com/spell=18755 [Improved Succubus - rank 2]
http://old.wowhead.com/spell=18756 [Improved Succubus - rank 3]
http://old.wowhead.com/spell=6358 [Seduction]

Rev:a29044373877b5682591
TDB: 335.11.40 Unix
Addons: Anticheat2 , AHbot, TrinityChat2

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/66271-warlock-talents-improved-succubus?utm_campaign=plugin&utm_content=tracker%2F1310&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F1310&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by FrancescoBorzi**
- closes #1543
- thanks @tabooo and @DevRival for code & test

### CODE REVIEW

**Comment by Shauren** [comment]
eh, no.

**Comment by DevRival** [comment]
In TC and so many hacks , this one is not what will not change , you have unsupported hacks but manages their mergers .

**Comment by ShinDarth** [comment]
The whole method seems a big hack to me. The whole spell system is a dark undocumented mess.

File: src/server/game/Spells/SpellMgr.cpp [hunk 1]
```diff
@@ -3756,6 +3756,21 @@ void SpellMgr::LoadSpellInfoCorrections()
                 break;
             // ENDOF ISLE OF CONQUEST SPELLS
             //
+            case 18754: // Improved succubus (1 talent)
+                spellInfo->Effects[EFFECT_0].ApplyAuraName = SPELL_AURA_ADD_FLAT_MODIFIER;     // affects the duration of seduction, let's minimize affection
```
Comment (jackpoz) [inline src/server/game/Spells/SpellMgr.cpp]: what are the current values ? the PR doesn't show the diff between old and new values for each modified spell

**Pull request closed without merging**